openid-4-verifiable-presentations-1_0.md

```diff
@@ -2249,6 +2247,45 @@ The VP Token contains the base64url-encoded `DeviceResponse` CBOR structure as d
 
 See ISO/IEC TS 18013-7 Annex B [@ISO.18013-7] and ISO/IEC 23220-4 Annex C [@ISO.23220-4] for the latest examples on how to use the `presentation_submission` parameter and how to generate the Authorizaton Response for presenting Credentials in the mdoc format.
 
+### `Handover` and `SessionTranscript` Definitions
+
+#### Invocation via the Digital Credentials API
+
+If the presentation request is invoked using the Digital Credentials API, the `SessionTranscript` CBOR structure as defined in Section 9.1.5.1 in [ISO.18013-5] MUST be used with the following changes:
+
+* `DeviceEngagementBytes` MUST be `null`.
+* `EReaderKeyBytes` MUST be `null`.
+* `Handover` MUST be the `OID4VPDCAPIHandover` CBOR structure as defined below.
+
+```cddl
+OID4VPDCAPIHandover = [
+  "OID4VPDCAPIHandover", ; A fixed identifier for this handover type
+  OID4VPDCAPIHandoverInfoHash ; A cryptographic hash of OID4VPDCAPIHandoverInfo
+]
+OID4VPDCAPIHandoverInfoHash = tstr  ; UTF-8 encoded string for the base64-encoded sha-256 hash of OID4VPDCAPIHandoverInfo
+OID4VPDCAPIHandoverInfo = [
+  origin,
+  client_id,
+  nonce
+] ; Array containing handover parameters
+client_id = tstr  ; UTF-8 encoded string
+origin = tstr    ; UTF-8 encoded string
+nonce = tstr     ; UTF-8 encoded string
+```
+
+The `OID4VPDCAPIHandover` structure has the following elements:
+
+- The first element MUST be the fixed UTF-8 encoded string `"OID4VPDCAPIHandover"`. This serves as a unique identifier for the handover structure to prevent misinterpretation or confusion.
+- The second element MUST be the `OID4VPDCAPIHandoverInfoHash`, represented as a UTF-8 string. This string encodes the base64-encoded (as defined in Section 4 of [@!RFC4648] -- not base64url-encoded) sha-256 hash of the `OID4VPDCAPIHandoverInfo` CBOR array, e.g., `H8BRh8j48O9oYatfu5AZzq6A9RINhZO5H16dQZngK7T62em8MUt1FLm52t+eX6xO`.
```

Comment:
Is there any benefit in encoding w/ base64url here or should we just use the raw Hash?
```cddl
ID4VPDCAPIHandover = [
  "OID4VPDCAPIHandover", ; A fixed identifier for this handover type
  OID4VPDCAPIHandoverInfoHash ; A cryptographic hash of OID4VPDCAPIHandoverInfo
]
OID4VPDCAPIHandoverInfoHash = bstr ; sha-256 hash of OID4VPDCAPIHandoverInfo
```

Reply:
No real benefit, just consistency reasons. I thought having UTF-8 strings consistently would reduce some encoding issues in implementations. I updated the PR to use bstr instead of tstr.

Reply:
Hash is using the raw hash, no longer W3C SRI.